CBOnlineIos/Presentation/otpVerify.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  otpVerify.swift
+//  CBOnlineIos
+//
+//  Created by Vaibhav Bisht on 09/07/20.
+//  Copyright © 2020 Coding Blocks. All rights reserved.
+//
+
+import UIKit
+
+extension UIViewController{
+    func hideKeyboradOtp() {
+        let tap :UITapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(dismissKeyboardOtp))
+        view.addGestureRecognizer(tap)
+    }
+    @objc func dismissKeyboardOtp() {
+        view.endEditing(true)
+    }
+}
+
+class otpVerify: UIViewController {
```

Comment:
`class OTPVerify` rename class and file name
In general class always with a capital letter